crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2033,19 +2033,26 @@ impl SymbolicationActor {
                     Duration::from_secs(60),
                     "minidump.stackwalk_new.spawn.error",
                     &minidump,
-                    diagnostics_cache,
+                    &diagnostics_cache,
                 ) {
                     Ok(stackwalking_result_new) => {
-                        if stackwalking_result_new.module_list
-                            == stackwalking_result_old.module_list
-                            && stackwalking_result_new.stacktraces
-                                == stackwalking_result_old.stacktraces
-                            && stackwalking_result_new.minidump_state
-                                == stackwalking_result_old.minidump_state
-                        {
-                            metric!(counter("minidump.stackwalk.results") += 1, "equality" => "equal")
-                        } else {
-                            metric!(counter("minidump.stackwalk.results") += 1, "equality" => "not equal")
+                        if stackwalking_result_new != stackwalking_result_old {
+                            Self::save_minidump(&minidump, &diagnostics_cache)
+                                .map_err(|e| log::error!("Failed to save minidump {:?}", &e))
+                                .map(|r| {
+                                    if let Some(path) = r {
+                                        sentry::configure_scope(|scope| {
+                                            scope.set_extra(
+                                                "crashed_minidump",
+                                                sentry::protocol::Value::String(
+                                                    path.to_string_lossy().to_string(),
+                                                ),
+                                            );
+                                        });
+                                    }
+                                })
+                                .ok();
+                            sentry::capture_error(&*anyhow!("Different stackwalking results"));
```

Comment:
This is pretty minor btw, and I don't really mind either way.  But I don't think we use the `anyhow!` macro at all yet, usually using `anyhow::Error::msg` for static strings.